lib/ExtUtils/MM_Win32.pm

```diff
@@ -395,8 +395,11 @@ sub perl_script {
 }
 
 sub can_dep_space {
-    my $self = shift;
-    1; # with Win32::GetShortPathName
+    # can_dep_space via GetShortPathName, if short paths are supported
```

Comment:
This needs a `require Win32;` for robustness.

Reply:
I have not gotten the knack of squashing commits yet...

Reply:
Can you make the indentation be exactly the same as the next line? I think you have a tab character.

Reply:
I've updated the PR. Please could you check - and let me know if you'd like me to squash the commits.

Reply:
This now looks fine, but yes - please squash the commits. Else the merger can do a squash, but it's better practice by far to have one commit be a small, meaningful change, not a partial tweak of a previous commit.

Reply:
Now squashed.